integration-cli/docker_cli_inspect_test.go

```diff
@@ -463,5 +463,5 @@ func (s *DockerSuite) TestInspectInvalidReference(c *check.C) {
 	// This test should work on both Windows and Linux
 	out, _, err := dockerCmdWithError("inspect", "FooBar")
 	c.Assert(err, checker.NotNil)
-	c.Assert(out, checker.Contains, "invalid reference format: repository name must be lowercase")
+	c.Assert(out, checker.Contains, "Error: No such object: FooBar")
```

Comment:
```
$ docker inspect FooBar
[]
Error response from daemon: no such image: FooBar: invalid reference format: repository name must be lowercase
```

Reply:
@cpuguy83 Discussed with @tiborvass @tonistiigi, this error should not be returned with detailed `no such image: FooBar: invalid reference format` message. Hence, it's already fixed in docker/cli#1071

Reply:
Actually, this test case will fail because the current release does not include the fix from cli yet. Can we still disable this test for now? @tiborvass @thaJeztah

Reply:
https://github.com/docker/cli/pull/1072 is merged, so I suggest we simply remove this test.

Reply:
@tiborvass resolved